Unknown Type Name ‘CFE_TBL_BufferSelect_Enum_t’

**Describe the bug**
SC is failing when building the app target due to unknown type name as seen here: https://github.com/arielswalker/SC/actions/runs/12756981276. 

```
/home/runner/work/SC/SC/cfe/modules/tbl/config/default_cfe_tbl_msgdefs.h:57:5: error: unknown type name ‘CFE_TBL_BufferSelect_Enum_t’
   57 |     CFE_TBL_BufferSelect_Enum_t ActiveTableFlag;       /**< \brief #CFE_TBL_BufferSelect_INACTIVE=Inactive Table,
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/runner/work/SC/SC/cfe/modules/tbl/config/default_cfe_tbl_msgdefs.h:78:5: error: unknown type name ‘CFE_TBL_BufferSelect_Enum_t’
   78 |     CFE_TBL_BufferSelect_Enum_t ActiveTableFlag;       /**< \brief #CFE_TBL_BufferSelect_INACTIVE=Inactive Table,
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [apps/sc/unit-test/CMakeFiles/coverage-sc_internal-stubs.dir/build.make:121: apps/sc/unit-test/CMakeFiles/coverage-sc_internal-stubs.dir/stubs/sc_cmds_stubs.c.o] Error 1
```

Due to this PR: https://github.com/nasa/cFE/pull/2459

**To Reproduce**
Follow steps in https://github.com/nasa/cFS/blob/main/.github/workflows/unit-test-coverage.yml 

**Expected behavior**
SC can build successfully in the Run unit test and coverage workflow. 

**Code snips**
CFE_TBL_BufferSelect_Enum_t is defined in https://github.com/nasa/cFE/blob/d6e610756cc8f2caddf465a92dff92fcd26d8e33/modules/tbl/config/default_cfe_tbl_extern_typedefs.h#L53 which is used in https://github.com/nasa/cFE/blob/d6e610756cc8f2caddf465a92dff92fcd26d8e33/modules/tbl/config/default_cfe_tbl_msgdefs.h#L57 The  default_cfe_tbl_msgdefs.h file does not define default_cfe_tbl_extern_typedefs.h.

**System observed on:**
GitHub workflows. 

**Additional context**
Add `#include "cfe_tbl_extern_typedefs.h"` in modules/tbl/config/default_cfe_tbl_msgdefs.h.

**Reporter Info**
Ariel Walker, MCSG TECH.

--- a/modules/tbl/config/default_cfe_tbl_msgdefs.h
+++ b/modules/tbl/config/default_cfe_tbl_msgdefs.h
@@ -30,6 +30,7 @@
 #include "cfe_mission_cfg.h"
 #include "cfe_es_extern_typedefs.h"
 #include "cfe_time_extern_typedefs.h"
+#include "cfe_tbl_extern_typedefs.h"
 #include "cfe_tbl_fcncodes.h"
 
 /********************************/

